python/ray/train/constants.py

```diff
@@ -74,3 +77,12 @@
 # By default these will be filtered out from ``train.report()``.
 # See ``TrainingCallback._preprocess_results`` for more details.
 ALL_RESERVED_KEYS = {PYTORCH_PROFILER_KEY}
+
+# Key to differentiate the `train_dataset` from other datasets
+TRAIN_DATASET_KEY = "_train_dataset"
+
+# Key to denote the preprocessor in the checkpoint dict.
+PREPROCESSOR_KEY = PREPROCESSOR_KEY
+
+# Key to denote the model in the checkpoint dict.
+MODEL_KEY = MODEL_KEY
```

Comment:
Fwiw, if you import them above you don't need to specify them again here (import will still work).

But why don't we just import from ray.ml.constants?

Reply:
Changed to just using ray.ml.constants!